README.md

```diff
@@ -105,6 +105,7 @@ Take a look at how this pattern may be applied in the table below.
 | `getPostData` | | `get` | `Post` | `Data` |
 | `handleClickOutside` | | `handle` | `Click` | `Outside` |
 | `shouldDisplayMessage` | `should` | `Display` | `Message`| |
+| `canAllowAnonymousUser` | `can` | `Allow` | `Anonymous`| `User` |
```

Comment:
I find `can*` variables generally suitable to be substituted with the verb itself as if cast from `it`:

```diff
-canAllowAnonymoursUser
+allowsAnonymousUser
```

Reply:
Cool! Makes sense.